packages/devextreme/js/__internal/core/state_manager/logger.ts

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-console */
+import type * as StateManagementTypes from './types';
+
+const LOG_TYPE_TO_LEVEL: Record<StateManagementTypes.LogLevel, number> = {
+  debug: 0, info: 1, warn: 2, error: 3,
+};
+
+export interface ConsoleLoggerOptions {
+  logLevel?: StateManagementTypes.LogLevel;
+  prefix: string;
+}
+
+export class Logger implements StateManagementTypes.Logger {
+  private logLevel: StateManagementTypes.LogLevel = 'info';
+
+  private prefix;
+
+  constructor(options?: ConsoleLoggerOptions) {
+    if (options) {
+      if (options.logLevel) {
+        this.logLevel = options.logLevel;
+      }
+      if (options.prefix !== undefined) {
+        this.prefix = options.prefix;
+      }
+    }
+  }
```

Comment:
Let's refactor this part a little bit

```suggestion
  private logLevel: StateManagementTypes.LogLevel;
  private prefix: string;

  constructor(options?: ConsoleLoggerOptions) {
    this.logLevel = options?.logLevel ?? 'info';
    this.prefix = options?.prefix ?? '';
  }
```

Reply:
Changed